web/portal/user/src/store/userStatus/status.ts

```diff
@@ -0,0 +1,96 @@
+import axios from 'axios';
+import { Action, action, Thunk, thunk } from 'easy-peasy';
+
+import { AppStore } from '../index';
+
+export interface Status {
+  companyDomain: string;
+  companyName: string;
+  extensionNumber: string;
+  gsQRCode: boolean;
+  ipRegistered: string;
+  language: string;
+  terminalName: string;
+  terminalPassword: string;
+  userAgent: string;
+  userName: string;
+  voiceMail: string;
+  features: Array<
+    | 'queues'
+    | 'recordings'
+    | 'faxes'
+    | 'friends'
+    | 'conferences'
+    | 'billing'
+    | 'invoices'
+    | 'progress'
+    | 'residential'
+    | 'wholesale'
+    | 'retail'
+    | 'vpbx'
+  >;
+}
+
+export type StatusState = {
+  profile: Status | null;
+};
+
+interface StatusActions {
+  setProfile: Action<StatusState, Status>;
+  resetProfile: Action<StatusState>;
+  init: Thunk<StatusState>;
+  load: Thunk<StatusStore, undefined, unknown, AppStore>;
+}
+
+export type StatusStore = StatusActions & StatusState;
+
+const Status: StatusStore = {
+  profile: null,
+  // actions
+  setProfile: action<StatusStore, Status>((state, profile) => {
+    state.profile = profile;
+  }),
+  resetProfile: action<StatusStore>((state) => {
+    localStorage.removeItem('profile');
+    state.profile = null;
+  }),
+  init: thunk<StatusStore>(async (actions) => {
+    const profile = localStorage.getItem('profile') as string | undefined;
```

Comment:
I don't think we need to store this at all

Reply:
It doesn't work without it.